example/dev-example/src/browser/api-test-menu.ts

```diff
@@ -892,6 +931,16 @@ export class ApiTestMenuContribution implements MenuContribution, CommandContrib
                     .then((response: any) => this.messageService.info(printResponse(response)));
             }
         });
+        commands.registerCommand(ValidationMarkersSuperBrewer3000JsonCommand, {
+            execute: () => {
+                this.modelServerClient
+                    .validation('SuperBrewer3000.json')
+                    .then((response: any) => {
+                        const message = createMarkersFromResponseBody(this.diagnosticManager, new URI('SuperBrewer3000.json'), response);
```

Comment:
In this test menu we use the `modelServerSubscriptionService` to react to updates from the server. I do believe it would be a nice use case to show that we can automatically update the markers if a validation result comes in. What do you think? Maybe we can provide a menu entry to activate/deactivate auto-marker creation which simply sets a boolean flag. We could then query the flag in `this.modelServerSubscriptionService.onValidationResultListener` (around line 1113) to decide we create markers or not. This is just an idea though, we don't have to do that in this PR.

Reply:
Personnaly, I deactivated the automatic validation. (see https://spectrum.chat/emfcloud/modelserver/does-model-server-support-emf-constraint-validation~4e3b41b7-aaf9-40ca-a509-bdc15f1966c8?m=MTYxMzk5NTA3MjE4Mg== )
This feature makes sense only for tiny models and kills performances for big models.
So I wouldn't work on this untill the automatic "liveValidation" vs EMF Live validation is better handled with possibilities to better control it.
Probably in another PR in the coming months... My goal though is not to achieve automatic validation with notifications, but rather have immediate feedbacks with JsonForms when EMF Live Validation fails (and the transaction is aborted).

Reply:
Right, I was only suggesting it for the dev-example API Test Menu cause it really helped me to see if everything was working as expected. As you mentioned, it is probably not useful (by default) as a dedicated feature. So we can definitely leave it as is for now.